app/migrations/versions/286d3cf3c133_.py

```diff
@@ -0,0 +1,40 @@
+"""Add transfer card usage table
+
+Revision ID: 286d3cf3c133
+Revises: 8e010839bc25
+Create Date: 2022-07-22 13:18:47.418917
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '286d3cf3c133'
+down_revision = '8e010839bc25'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('transfer_card_usage',
```

Comment:
Is `transfer_card_usage` the best name?

Reply:
@tristanhcole I suppose it is a bit confusing, since there's other `usage` verbiage in the platform. Maybe we can call it `transfer_card_state`? What do you think?

Reply:
I like `transfer_card_state` more